lib/commands/execute.js

```diff
@@ -84,6 +84,10 @@ extensions.executeMobile = async function executeMobile (mobileCommand, opts = {
     clearKeychains: 'mobileClearKeychains',
 
     getPermission: 'mobileGetPermission',
+    setPermission: 'mobileSetPermissions',
```

Comment:
What about `changePermissions` like Android?
(I also thought to keep `setPermissions` for iOS since the parameter was different. So, it is fine to go with current name, too.)